### DISCUSSION TITLE
Issue in Accessing GitHub Raw Files in All Repos

### DISCUSSION THREAD

**Discussion by piyush-multiplexer** [community/community#56223] [Repositories] [answered]
### Select Topic Area

General

### Body

I'm trying to access raw files in the GitHub repository but it has been out for the last 2 day. This is happening for all the repositories. If I try with VPN it works well but with LAN and mobile data it fails. Due to this there are certain issues in tool setup/installation as raw file dependencies.

I have tried the URLs below but the same problem occurs.
- https://raw.githubusercontent.com/nuxt/starter/templates/templates/v3.json
- https://github.com/github/vscode-github-actions/raw/main/README.md

Does it happen based on GeoGraphic Area as not accessible in mobile/data and laptop/LAN?

**Comment by airtower-luna** [comment, marked as answer]
Your profile says you're in India, and there have been a lot of people from India reporting their provider blocking the domain:

* #32889

If it's that, using an alternative DNS server (e.g. [Quad9](https://quad9.net/service/service-addresses-and-features/)) might help.

**Reply by piyush-multiplexer** [reply]
Yeah it seems an ISP side problem. Working after changing DNS.
